spec/support/a_framework_assessment.rb

```diff
@@ -346,23 +355,6 @@
       expect { assessment.build_responses! }.to raise_error(ActiveRecord::RecordInvalid)
     end
 
-    it 'does not persist any responses if there are any invalid assessment records' do
```

Comment:
this test isn't valid anymore as we will never get a wrong type of response, as we are explicitly defining those values ourselves